slurm/docs/guides/05-run-jobs.md

```diff
@@ -0,0 +1,140 @@
+graphical jobs" guide ([run-graphical-jobs](run-graphical-jobs)).
+
+However, some applications are non-interactive and you are able to
+pass them a number of command-line parameters (for example input and
+output files) and they then run for a number of minutes / hours / days
+processing data.
+
+## THE SCHEDULER
+
+All jobs to be executed within the compute environment must be
+submitted through the job scheduler. This ensures fair usage as well
+as making sure your job gets the best of the available resources
+allocated to it. Whilst you can login to the compute nodes directly to
+check on your jobs (via `ssh`), any processes on compute nodes that
+are run outside of the scheduler are automatically swept periodically.
+
+When submitting new job requests to the scheduler, you should make
+your best guess for the resources it will require. If you find the job
+gets terminated, increase your resource request to allow your job to
+run. It's often worth starting with the defaults as these are usually
+sufficient for most applications and then alter them if you have
+problems. 
+
+Requesting resources as close to your jobs requirements as possible
+will ensure that your job is scheduled as quickly as possible. Jobs
+that request more resource will potentially queue for longer as they
+need to wait for more resources to become available. The scheduler can
+use jobs with smaller resource requests to backfill if it knows that
+the smaller job will complete before enough resources will become
+available for a larger, potentially higher priority, job to
+start. This allows users requesting the correct resources &mdash; in
+particular `--time` (runtime) and `--mem` (maximum memory usage)
+&mdash; to jump the queue and start their jobs more quickly.
+
+## INTERACTIVE JOBS
+
+Interactive jobs can be run via the scheduler using the interactive
+shell command `srun`. Running `srun` with the required options
+will allocate you a shell on the chosen compute node. From there,
+you can then load your required application modules and start running.
+
+Often when experimenting with a new application it can be useful to
+start an interactive session to understand the way it behaves and the
+command line options available before you write a job script.
+
+The following command can be used to gain an interactive session on
+a cluster compute node: 
+
+   `srun --pty /bin/bash`
+
+You can optionally choose to provide information to the scheduler
+about the interactive session you are creating - for example duration,
+maximum memory, number of CPU cores to allocate. These can be provided
+using some of the following options together with the above `srun`
+command.
+
+ * `--time=0-2:00`:
+
+   How long you will potentially keep the session open for (d-hh:mm);
+   the session will be terminated when this time is exceeded
+
+ * `--mem 512`:
+
+   How much memory you will potentially want to use in this session.
```

Comment:
Add "in megabytes" to explain units (if it is megabytes, looks like it given the rest of the copy in this paragraph).